commit 45c09bfe58c37bbf7965af25bdd4fa5c37c0908f
Author: Rodrigo Vivi <rodrigo.vivi@gmail.com>
Date:   Wed Jun 5 13:07:13 2013 -0300

    Add more correct names for Haswell.
    
    As we find out more of the final product names for Haswell chipsets, we
    need to update the user visible identification strings.
    
    Signed-off-by: Rodrigo Vivi <rodrigo.vivi@gmail.com>

diff --git a/src/intel_module.c b/src/intel_module.c
index 8b3b196..c1d0e09 100644
--- a/src/intel_module.c
+++ b/src/intel_module.c
@@ -160,19 +160,19 @@ static const SymTabRec intel_chipsets[] = {
 	{PCI_CHIP_IVYBRIDGE_D_GT2,		"Ivybridge Desktop (GT2)" },
 	{PCI_CHIP_IVYBRIDGE_S_GT1,		"Ivybridge Server" },
 	{PCI_CHIP_IVYBRIDGE_S_GT2,		"Ivybridge Server (GT2)" },
-	{PCI_CHIP_HASWELL_D_GT1,		"Haswell Desktop (GT1)" },
-	{PCI_CHIP_HASWELL_D_GT2,		"Haswell Desktop (GT2)" },
+	{PCI_CHIP_HASWELL_D_GT1,		"HD Graphics" },
+	{PCI_CHIP_HASWELL_D_GT2,		"HD Graphics 4600" },
 	{PCI_CHIP_HASWELL_D_GT3,		"Haswell Desktop (GT3)" },
-	{PCI_CHIP_HASWELL_M_GT1,		"Haswell Mobile (GT1)" },
-	{PCI_CHIP_HASWELL_M_GT2,		"Haswell Mobile (GT2)" },
+	{PCI_CHIP_HASWELL_M_GT1,		"HD Graphics" },
+	{PCI_CHIP_HASWELL_M_GT2,		"HD Graphics 4600" },
 	{PCI_CHIP_HASWELL_M_GT3,		"Haswell Mobile (GT3)" },
-	{PCI_CHIP_HASWELL_S_GT1,		"Haswell Server (GT1)" },
-	{PCI_CHIP_HASWELL_S_GT2,		"Haswell Server (GT2)" },
+	{PCI_CHIP_HASWELL_S_GT1,		"HD Graphics" },
+	{PCI_CHIP_HASWELL_S_GT2,		"HD Graphics P4600/P4700" },
 	{PCI_CHIP_HASWELL_S_GT3,		"Haswell Server (GT3)" },
 	{PCI_CHIP_HASWELL_B_GT1,		"Haswell (GT1)" },
 	{PCI_CHIP_HASWELL_B_GT2,		"Haswell (GT2)" },
 	{PCI_CHIP_HASWELL_B_GT3,		"Haswell (GT3)" },
-	{PCI_CHIP_HASWELL_E_GT1,		"Haswell (GT1)" },
+	{PCI_CHIP_HASWELL_E_GT1,		"HD Graphics" },
 	{PCI_CHIP_HASWELL_E_GT2,		"Haswell (GT2)" },
 	{PCI_CHIP_HASWELL_E_GT3,		"Haswell (GT3)" },
 	{PCI_CHIP_HASWELL_SDV_D_GT1,		"Haswell SDV Desktop (GT1)" },
@@ -193,23 +193,23 @@ static const SymTabRec intel_chipsets[] = {
 	{PCI_CHIP_HASWELL_ULT_D_GT1,		"Haswell ULT Desktop (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_D_GT2,		"Haswell ULT Desktop (GT2)" },
 	{PCI_CHIP_HASWELL_ULT_D_GT3,		"Iris(TM) Graphics 5100" },
-	{PCI_CHIP_HASWELL_ULT_M_GT1,		"Haswell ULT Mobile (GT1)" },
-	{PCI_CHIP_HASWELL_ULT_M_GT2,		"Haswell ULT Mobile (GT2)" },
-	{PCI_CHIP_HASWELL_ULT_M_GT3,		"Iris(TM) Graphics 5100" },
+	{PCI_CHIP_HASWELL_ULT_M_GT1,		"HD Graphics" },
+	{PCI_CHIP_HASWELL_ULT_M_GT2,		"HD Graphics 4400" },
+	{PCI_CHIP_HASWELL_ULT_M_GT3,		"HD Graphics 5000" },
 	{PCI_CHIP_HASWELL_ULT_S_GT1,		"Haswell ULT Server (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_S_GT2,		"Haswell ULT Server (GT2)" },
 	{PCI_CHIP_HASWELL_ULT_S_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_ULT_B_GT1,		"Haswell ULT (GT1)" },
 	{PCI_CHIP_HASWELL_ULT_B_GT2,		"Haswell ULT (GT2)" },
 	{PCI_CHIP_HASWELL_ULT_B_GT3,		"Iris(TM) Graphics 5100" },
-	{PCI_CHIP_HASWELL_ULT_E_GT1,		"Haswell ULT (GT1)" },
-	{PCI_CHIP_HASWELL_ULT_E_GT2,		"Haswell ULT (GT2)" },
+	{PCI_CHIP_HASWELL_ULT_E_GT1,		"HD Graphics" },
+	{PCI_CHIP_HASWELL_ULT_E_GT2,		"HD Graphics 4200" },
 	{PCI_CHIP_HASWELL_ULT_E_GT3,		"Iris(TM) Graphics 5100" },
 	{PCI_CHIP_HASWELL_CRW_D_GT1,		"Haswell CRW Desktop (GT1)" },
-	{PCI_CHIP_HASWELL_CRW_D_GT2,		"Haswell CRW Desktop (GT2)" },
+	{PCI_CHIP_HASWELL_CRW_D_GT2,		"HD Graphics 4600" },
 	{PCI_CHIP_HASWELL_CRW_D_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_M_GT1,		"Haswell CRW Mobile (GT1)" },
-	{PCI_CHIP_HASWELL_CRW_M_GT2,		"Haswell CRW Mobile (GT2)" },
+	{PCI_CHIP_HASWELL_CRW_M_GT2,		"HD Graphics 4600" },
 	{PCI_CHIP_HASWELL_CRW_M_GT3,		"Iris(TM) Pro Graphics 5200" },
 	{PCI_CHIP_HASWELL_CRW_S_GT1,		"Haswell CRW Server (GT1)" },
 	{PCI_CHIP_HASWELL_CRW_S_GT2,		"Haswell CRW Server (GT2)" },
